nebula_common/include/nebula_common/nebula_common.hpp

```diff
@@ -447,11 +455,31 @@ inline std::ostream & operator<<(std::ostream & os, nebula::drivers::SensorModel
 struct SensorConfigurationBase
 {
   SensorModel sensor_model;
-  ReturnMode return_mode;
+  std::string frame_id;
+};
+
+/// @brief Base struct for Ethernet-based Sensor configuration
+struct EthernetSensorConfigurationBase : SensorConfigurationBase
+{
   std::string host_ip;
   std::string sensor_ip;
-  std::string frame_id;
   uint16_t data_port;
+};
+
+/// @brief Base struct for Ethernet-based Sensor configuration
+struct CANSensorConfigurationBase : SensorConfigurationBase
+{
+  std::string interface;
+  float receiver_timeout_sec{};
+  float sender_timeout_sec{};
+  std::string filters{};
```

Comment:
Please add a doc comment `///` specifying format requirements and/or an example

Reply:
Added some information in [85cac29](https://github.com/tier4/nebula/pull/124/commits/85cac29c5b0831d6d00dc30657828875779907cc)